apps/www/lib/rss.tsx

```diff
@@ -22,7 +22,10 @@ const generateRssItem = (post: any): string => {
   const encodedTitle = xmlEncode(post.title)
   const encodedPath = xmlEncode(post.path)
   const encodedDescription = xmlEncode(post.description)
-  const formattedDate = dayjs(post.date).utc().format('ddd, DD MMM YYYY HH:mm:ss [GMT]')
+  const formattedDate = dayjs(post.date)
+    .utcOffset(0, true)
```

Comment:
should we offset it to California time?

Reply:
-7

Reply:
sure!

Reply:
i'm not sure what is right or wrong here, we probably need to standardize the times in the blog posts as well